packages/dds/tree/src/change-family/README.md

```diff
@@ -0,0 +1,14 @@
+# change-family
+
+Abstraction for change families.
+
+A change family is a collection of types of changes which can be used within a document, and support for building, serializing and rebasing them over each-other.
+This includes things like policies for how to edit sequences, clearing optional fields, swapping trees, updating counters, schema editing etc.
+
+A change family has to handle all its supported changes identically between all clients, and across all versions.
+Support for new kinds of changes can be added, but updating applications to use the new changes must be done carefully to avoid breaking old clients.
+
+Applications that only have ephemeral sessions and never persist documents with trailing ops may be able to relax this constraint somewhat.
```

Comment:
What are trailing ops?

Reply:
Ops stored in server side summaries which are applied on container load. Thet exist to prevent dataloss when a session ends between summaries. As they are an important concept in fluid, but we don't seem to define them anywhere (though the term is used several times), I'm asking internally where documentation about them belongs and will try and update and link that.

Reply:
Looks like there currently arn't docs covering trailing ops. When such docs get added I'll update this to link them, but for now I'll leave this as is.